arviz/tests/test_data.py

```diff
@@ -11,7 +11,11 @@
     from_pymc3,
     from_pystan,
     from_emcee,
+    load_arviz_data,
+    list_datasets,
+    clear_data_home,
 )
+from arviz.data.datasets import REMOTE_DATASETS, LOCAL_DATASETS, RemoteFileMetadata
```

Comment:
Consistency for absolute vs relative imports in ArviZ would be nice but not required.